Documentation/csi_block_storage_kc_whatsnew.md

```diff
@@ -0,0 +1,8 @@
+# What's new
```

Comment:
consider using inner directories.
I recommend using the same dir structure as the table of contents hierarchy, but if it's too much, then at least move all non-summary.md files to some inner, flat dir, so users would find it easily.